static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
```

Comment:
Moving `removeContact` to be a method within `ContactsCtrl` seems to work, even if it's assigning `$scope.items`.  I will push this change, but please check if I'm missing something and it's not working as it should.